evm/src/NttManager/NttManager.sol

```diff
@@ -30,6 +30,9 @@ import {ManagerBase} from "./ManagerBase.sol";
 ///  - the amount
 ///  - the recipient chain
 ///  - the recipient address
+///  - the refund address: the address to which refunds are issued for any unused gas
+///    for attestations on a given transfer. Currently, the gas limit is configured
```

Comment:
I would say, "If the gas limit is configured too high" since the integrator decides what gas limit to pass to the constructor.